content/work/adobe.md

```diff
@@ -39,4 +96,149 @@ summary: |
+
+[type tasting]: https://www.typetasting.com/
+
+<figure>
+<img src="{{ site.images }}work/adobe/enter-here.png" class="extend-full" alt="" /><figcaption>The vintage Ferris wheel
+went through a sketch phase and a couple of drafts before
+the design was finalized.</figcaption>
+</figure>
+<figure>
+<img src="{{ site.images }}work/adobe/edgy.png" class="extend-full" alt="" /><figcaption>Near the end of the edgy superpower design, the team realized the word "edgy" was hard to read in the lowercase Totally Gothic typeface.</figcaption>
+</figure>
+
+
+### Audio & Nuxt.js
+
+Audio implementation provided an interesting challenging. First, we tried Web Audio API and a Nuxt.js plugin that automatically played and paused the sounds on each page of the game. When we tested it, the sounds played simultaneously. Uh oh! We switched to HTML5 Audio and added an ID for each question tag. The correct sound played when the ID and tag matched. Yay!
+
+During testing across different browsers and devices, we discovered that our audio solution didn’t work correctly everywhere — returning auto-play errors. We had to get clever. Using [HTML5 audio] made recovery trivial if autoplay failed.
+
+```js
+controlSound(on) {
+  const bg = this.$refs.audioBackground;
+  if (bg) {
+    on ? bg.play().catch(() =>  
+    this.toggleSetting(this.sound)) : bg.pause();
+  }
+},
+```
+
+It was fun collaborating with audio specialist, Rob Taliesin Owen of [Potion SoundWorks]. Adding sound elevated the whole experience of game play. 
+
+[html5 audio]: https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide
+[Potion SoundWorks]: https://potionsoundworks.com/
+
+### CSS Custom Properties
+
+Each question in the game has nine options and each option consists of the same identical image plus a unique font. Typefaces have a wide array of letter heights and widths. When we placed the typefaces on top of the images, some barely filled the space, some hung off the edges of the illustrations. CSS Custom Properties to the rescue! Custom Properties made it possible to adjust individual typefaces.
+
+{{ embed.figure(
+  data=before,
+  caption='A default font-size does not work well with different typefaces'
+) }}
+
+First, we add the `svg-adjust` custom property with js to the outer container 
+of each illustration.
+
+```js
+computed: {
+  fontStyles() {
+    return {
+      '--svg-adjust': this.font.adjust,
+    };
+  }
+}
+```
+
```

Comment:
Other than spelling out javascript, is there a better or more correct way to say what is happening in this code sample/on this project @duggiemitchell @mirisuzanne @jgerigmeyer (whoever gets here first?)